### ISSUE TITLE
Slow Cutscenes

### ISSUE THREAD

**Issue by geo98** [RPCS3/rpcs3#3496]
On some games like Dantes Inferno the cutscenes plays very slow with audio stutter.

I think its and old and known issue but since nobody has ever opened this issue before, I decided to open it until it gets totally fixed.

Many thanks for everyone.

(Edit: I'm attaching my save data, so you will have skipable cutscenes when create a new game.)

[PS3 Dantes Inferno Save Data.zip](https://github.com/RPCS3/rpcs3/files/1327022/PS3.Dantes.Inferno.Save.Data.zip)

**Comment by Vlakipn** [comment]
Yes, this is a big dealbreaker if you dont have save file or you want to actualy watch cutscenes.
In addition, audio interrupts also in cutscenes.

**Comment by zminhquanz** [comment]
Persona 5 it still stutter and audio interrupt when you use slower CPU , although on title screen FPS 60

**Comment by ThatFlint** [comment]
Same here with Persona 5, this is a very annoying and grating issue. Watching the cutscenes hurts my ears :c

**Comment by kennedyoliveira** [comment]
Is there a workaround for it? I have the same issues

**Comment by marcussacana** [comment]
Here, it's better when enable the vulkan + 2 SPU Threads., if you have a i3 or lower, maybe 1 Thread.

**Comment by VelocityRa** [comment]
Are they pre-rendered or in-engine?

**Comment by kennedyoliveira** [comment]
hey @marcussacana i tried with 2 SPU threads no luck, by the way, i got a Ryzen 7 1700 @ 4.0 on 8 cores

**Comment by Asinin3** [comment]
@kennedyoliveira https://www.youtube.com/watch?v=aZ_HwioDhI0

**Comment by kennedyoliveira** [comment]
@Asinin3 thanks for the info! I'll try it and post the results back here

**Comment by braedonmartin** [comment]
Here's the fix for the audio stutter in Persona 5, might help with other games as well: https://www.reddit.com/r/rpcs3/comments/7nnrfq/how_i_made_persona_5_playable_on_rpcs3_audio/

**Comment by ianhyzy** [comment]
the fix didn't work for me on a skylake i5

**Comment by ikki84** [comment]
@VelocityRa Afaik it's pre-rendered. 
-A similar behaviour happens in BLES02014. during the intro's movie. The fps counter is locked at 30 almost all the time, just dropping to 28-27, but if you pay attention it is clearly stuttering some parts of the movie, including video and audio. 
-Changing the video rendering/backend to Null doesn't made the audio stuttering reduce at all.

I know, this discussion about, "That game 5" audio stuttering is becoming annoying, but doesn´t it feel strange that playing a cutscene  "a video file", must be that demanding at all???

**Comment by rodjun** [comment]
Not really, decoding videos is a very cpu intensive, and probably done on SPUs which aren't extremely fast on rpcs3

**Comment by rodjun** [comment]
Not really, decoding videos is a very cpu intensive, and probably done on SPUs which aren't extremely fast on rpcs3

**Comment by JamieeLee** [comment]
work around for the audio is disable bind spu detection and just enable bind spu's to secondary cores then lower there thread priority, typically on a i7 2-3 threads on i7 and 1 -2 max on i 5 ... Tested with persona 5 for 8 hours no audio issues present in all but one cut scene which could just been  partial bad dump on my end.   DQB, Catherine no audio in cut scene videos at all.

**Comment by ikki84** [comment]
https://github.com/RPCS3/rpcs3/pull/4108 Improves reasonably the performance, at least in the cutscenes that were stuttering in my tests. Once stable and merged this issue could be closed imo...

**Comment by oangelo** [comment]
Same here with Dante's Inferno. The FPS is bigger than 100, but the cutscenes are slow and the audio is stuttering. I have tried many different configs, and nothing seems to improve. And my machine is a Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz | 12 Threads | 7.75 GiB RAM | AVX+ | TSX!

**Comment by maximilian578** [comment]
The issue still persists for prerendered cutscenes. Try disabling SPU loop detection. Works for me on Persona 5. It does lower performance but after #5435 is merged it will be a small problem.

**Comment by Illynir** [comment]
No problem for me for months now. Dante Inferno's cutscene are smooth even on my i5.

I think this issue can be closed.

**Comment by mrdratik** [comment]
I don't know why this issue is marked as resolved, because I'm experiencing it right now in Dante’s Inferno. The gameplay runs at a smooth 60 fps, but the pre-rendered cutscenes play at half speed, are constantly out of sync with the audio, and the audio plays in fits and starts

**Comment by AniLeo** [comment]
This ticket is 7 years old and was clearly closed due to the lack of repro. File a new ticket following the template.
